test/Makefile

```diff
@@ -0,0 +1,36 @@
+FILE = function-names
+MINFILE = $(FILE).min
+
+# wasm function name and arguments
+FUNCTION = _Z4add1i
+ARGS = 10
```

Comment:
Eventually we may want to capture these things from the analysis. But right now, for the Makefile, this is fine.